project-document/docs/specifications/others.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Others
+---
+
+## Timezone
+
```

Comment:
Something like this make sense more, what do you think?
`Follow the ISO format, including the time zone.`

Reply:
I've added mentioning it.